nimbusPython/NimbusClient.py

```diff
@@ -398,7 +405,52 @@ def getLog(self):
         if (rv == 0):
             data = result["result"]
         return rv, data
+    
+    def setAmplitude(self, ampl):
+        result = self._getJSONParameter("AutoExposure", 2, None)
```

Comment:
I guess _getJSONParameter is not needed.

Reply:
@Coimbra1984 Warum wird das nicht benötigt? Du hast das bei setExposure ähnlich gedacht, habe angenommen das sei daher richtig. :D

Reply:
Bei setExposure wird ein Array übergeben. Damit ich die richtige Array Länge habe, mache ich vorher ein getExposure und überschreibe dann jeweils die Einträge:

result = self._getJSONParameter("nimbusRaw", 0, None)
rv = result["success"]
if rv == 0:
        seqs = result["result"]

Reply:
@Coimbra1984 Ah, danke! Ja, das macht sinn, wird gleich gefixt. :)